x-pack/solutions/security/plugins/security_solution_serverless/common/pli/pli_config.ts

```diff
@@ -22,8 +22,9 @@ export const PLI_PRODUCT_FEATURES: PliProductFeatures = {
       ProductFeatureKey.configurations,
       ProductFeatureKey.externalDetections,
     ],
-    essentials: [ProductFeatureKey.attackDiscovery, ProductFeatureKey.assistant],
-    complete: [ProductFeatureKey.attackDiscovery, ProductFeatureKey.assistant],
+    // neither of these tiers are available in ai_soc product line
+    essentials: [],
+    complete: [],
```

Comment:
Actually, the `ProductLine.aiSoc` add-on will be available for _essentials_ (not sure _complete_), right?